_data/simple-icons.json

```
@@ -11959,6 +11959,11 @@
             "hex": "E1171E",
             "source": "https://picnic.app/nl/feestdagen/"
         },
+        {
+            "title": "picoCTF",
+            "hex": "C41230",
+            "source": "https://picoctf.org/img/logos/picoctf-logo-horizontal-white.svg"
```

Use just the page where appears the logo.

```suggestion
            "source": "https://picoctf.org"
```